security/advancedtls/advancedtls.go

```diff
@@ -186,10 +186,17 @@ type ClientOptions struct {
 	// IdentityOptions is OPTIONAL on client side. This field only needs to be
 	// set if mutual authentication is required on server side.
 	IdentityOptions IdentityCertificateOptions
+	// AdditionalPeerVerification is a custom verification check after certificate signature
+	// check.
+	// If this is set, we will perform this customized check after doing the
+	// normal check(s) indicated by setting VType.
```

Comment:
VType->VerificationType?

Reply:
Ah I think I lost some in the merge, good catch